### DISCUSSION TITLE
Security Questions

### DISCUSSION THREAD

**Discussion by TalMcMahon** [OrchardCMS/OrchardCore#8299] [Q&A]
Is there already a method to add a security question answer to password reset? 
I suspect there is not, If that is the case, is there then a way to have a User edit their own Custom user settings? (I was thinking I could add it there, but that seems to only have admin availability)

Any other suggestions?
BTW, I am not looking for a "You shouldn't," my business owner insists it has to be part of the app.  

Thanks

**Comment by hishamco** [comment]
@sebastienros is there any official security channel, for posting a security related questions?
